mount-helper/README.md

```diff
@@ -0,0 +1,13 @@
+```
+Copyright (c) IBM Corp. 2023. All Rights Reserved.
+Project name: VPC File Storage Mount Helper
+This project is licensed under the MIT License, see LICENSE file in the root directory.
+```
+
+# Mount Helper
+Mount helper is an open source automation tool that helps to establish secure IPsec connection between customer virtual server instance (VSI) and storage server. The mount helper ensures that the communication between the customer VSI and the file share is encrypted in transit.
```

Comment:
This package is specific to *IBM Cloud VPC File Storage*

Reply:
yes

Reply:
Needs to be fixed, may be sometime soon.